relevanceai/operations_new/processing/text/count/base.py

```diff
@@ -0,0 +1,59 @@
+from relevanceai.operations_new.base import OperationBase
+
+
+class CountTextBase(OperationBase):
+    _imported_nltk = False
+
+    def __init__(
+        self,
+        text_fields: list,
+        include_char_count: bool = True,
+        include_word_count: bool = True,
+        include_sentence_count: bool = False,
+        output_fields: list = None,
+    ):
+        self.text_fields = text_fields
+        self.include_char_count = include_char_count
+        self.include_word_count = include_word_count
+        self.include_sentence_count = include_sentence_count
+        self.output_fields = output_fields
+
+    def _init_counting_sentences(self):
+        import nltk
+
+        nltk.download("punkt")
+        self._imported_nltk = True
+
+    def count_characters(self, text):
+        return len(text)
+
+    def count_words(self, text):
+        return len(text.split())
+
+    def count_sentences(self, text):
+        if not self._imported_nltk:
```

Comment:
is this the most reliable method, we could just use sentence_splitter or something.

Reply:
I'm not heavily opinionated here. my preference for `nltk` was so that users didn't have to install another thing but I haven't tested properly in comparison to sentence-splitter. I'll focus on using sentence-splitter for now as that is consistent with past behavior and we know what to expect from that one.

Reply:
is nltk default required?